benchmark.cfg.example

```diff
@@ -20,7 +20,7 @@ query_levels=[1, 5,10,15,20]
 threads=8
 mode=benchmark
 os=linux
-server_host=127.0.0.1
+server_host=TFB-server
```

Comment:
This would be part of the setup. So it would be something done manually (see the linked installation guide) or did you have something else in mind?